cypress/support/index.js

```diff
@@ -1,4 +1,4 @@
-import '@bigtest/cypress';
+import '@testing-library/cypress/add-commands';
```

Comment:
I had to remove 
```js
import '@bigtest/cypress';
```
because it broke the `.should(chainers)` assertions like
```js
cy.findByRole('dialog').should('not.exist');
```

Reply:
⚠️ This breaks the tests that use interactors!

Reply:
Interesting. How did it break `.should(chainers)`? In other words, what is the expected behavior, and what is the actual behavior?

Reply:
I created an issue to track this https://github.com/thefrontside/bigtest/issues/962

Reply:
`.should(chainers)` gave me an error, until I removed the Bigtest import here.

Took me considerable time to locate the thing BTW )

Reply:
Sorry for the trouble and thank you for reporting it. We'll look into this ASAP.

Reply:
@cowboyd it looks like @minkimcello tracked it down https://github.com/thefrontside/bigtest/issues/962#issuecomment-891153757

Reply:
I've published a preview package with cypress bumped to `7.7.0` so if anyone would like to give it a try, please update your `package.json` and update the version of `@bigtest/cypress` to `mk_cypr770`:
```
{
  "dependencies": {
    "@bigtest/cypress": "mk_cypr770"
  }
}
```

Reply:
@minkimcello should cypress be a peerDependency for `@bigtest/cypress`?

Reply:
@taras I think it should be. I only have it as a dependency because someone thought users should only have to install `@bigtest/cypress` instead of `@bigtest/cypress` _and_ `cypress`.

Reply:
@minkimcello yeah, it looks like it would have prevented this issue

Reply:
@axelhunn @taras @minkimcello 

I've tracked the core issue down to a problem with the way cypress is consuming modules internally. My guess is that it has to do with how cypress was historically bundled and written in coffee script.

https://github.com/cypress-io/cypress/issues/17564 


However, given the number of open issues on the cypress repository (`1.5k`) I don't think we can count on it getting attention any time soon. That means we either need to rename the `expect` method to something like `assert`, or try to do some deep magic workaround.